### PR TITLE
Feat/homepage

### LINKED ISSUE #1: Landing page should be implemented as designs
**What:** Landing page should be implemented as designs

**How:**

- Using a CSS framework implement designs to be rendered by Rails.

**Notes**:

- Designs are not final yet, I'll update this ticket with links once it's done

### LINKED ISSUE #2: Landing page has SEO tags and previous had been tested correctly
**What:** Landing page has SEO tags and previous had been tested correctly

**How:**

- Research a way to add SEO tags to the landing page
- Test with Open Graph debugging tools for al major social media platforms

### LINKED ISSUE #3: Landing Page

### PR DESCRIPTION
**Pull request by castrolem**
**What:**
Add homepage with plans and donation box

**Why:**
To have a landing page where users can see the benefits of donating.
Closes #1, closes #2 and closes #3

**How:**
By creating a static rendered page that loads the plans through react components.

### CODE REVIEW

**Comment by castrolem** [comment]
## Screenshots
For some reason, I cannot take a full-page desktop screenshot with Chrome, it stretches the window and does not represent the actual sizing.

localhost_3000_(iPhone 5_SE)

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+ *
+ * You're free to add application-wide styles to this file and they'll appear at the bottom of the
+ * compiled file so the styles you add here take precedence over styles defined in any other CSS/SCSS
+ * files in this directory. Styles in this file should be added after the last require_* statement.
+ * It is generally better to create a new file per style scope.
+ *= require_tree .
+ *= require_self
+ */
+
+html,
```
Comment (orlando) [inline app/assets/stylesheets/application.scss:16]: @castrolem How are we requiring all the files, I see we have a globals file, but what about using the `applications.scss` file to require all the other files in a certain order?

**application.scss**
```scss

@import 'globals';
@import 'footer';
...
```

**Review by orlando — CHANGES_REQUESTED** [review]

**Comment by orlando** [comment]
@castrolem there are also some issues regarding breakpoints.

image

Also, let's review the colors on a call, our background colors should look like the one we have in the community https://community.debtcollective.org/

**Comment by castrolem** [comment]
@orlando fixed comments

**Review by orlando — APPROVED** [review]
LGTM :+1:

**Pull request merged**